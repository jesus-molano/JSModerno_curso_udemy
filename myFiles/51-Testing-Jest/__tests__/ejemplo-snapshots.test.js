const cliente = {
  nombre: "Jesus Molano",
  balance: 500,
  tipo: "Premium",
};

describe("Testing al cliente", () => {
  test("Es Jesus Molano", () => {
    expect(cliente).toMatchSnapshot();
  });
});

// La primera ejecucion crea el snapshot
// Las siguientes veces comparara el objeto actual con el snapshot
// Para sobreescribir el snapshot => npm test -- -u
