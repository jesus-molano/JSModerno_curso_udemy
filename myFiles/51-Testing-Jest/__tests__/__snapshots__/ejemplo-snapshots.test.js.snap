// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing al cliente Es Jesus Molano 1`] = `
Object {
  "balance": 500,
  "nombre": "Jesus Molano",
  "tipo": "Premium",
}
`;
